addons/civilian/functions/fnc_getNearestVehicle.sqf

```diff
@@ -0,0 +1,30 @@
+#include "script_component.hpp"
+/*
+ * Author: 3Mydlo3
+ * Function returns nearest vehicle for given position.
+ *
+ * Arguments:
+ * 0: Position <POSITION/OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [player] call afsk_civilian_fnc_getNearestVehicle
+ *
+ * Public: No
+ */
+
+params ["_pos", ["_radius", 500]];
+
+if (_pos isEqualType objNull) then {
+    _pos = getPos _pos;
+};
+
+private _nearbyVehicles = _pos nearEntities [["Air", "Car", "Motorcycle", "Tank"], _radius];
+
+private _vehiclesSorted = [_nearbyVehicles, _pos] call BIS_fnc_nearestPosition;
+
+if (_vehiclesSorted isEqualTo [0, 0, 0]) exitWith {objNull};
+
+_vehiclesSorted param [0, objNull]
```

Comment:
IIRC `BIS_fnc_nearestPosition` returns a single element so that `param` is unnecessary.

Reply:
Yea, it seems like this function initially was done a bit differently with sorting. Fixed.